lib/nexmo_developer/app/assets/javascripts/digitalData.js

```diff
@@ -1,3 +1,5 @@
+const PRIMARY_CATEGORY = document.getElementById('adobe_launch_url').value
```

Comment:
this is the category not the launch URL - they are different things.

Reply:
This is corrected.